main/templates/main/about.html

```diff
@@ -0,0 +1,58 @@
+{% extends "main/base.html" %}
+{% block content %}
+<div class="row">
```

Comment:
For consistency with other pages let's wrap everything in a div with `class=ui text container`. If we do that we can also get rid of all of the div's with `class=row`.

Reply:
Thankd got highlighting this @cc-a, HTML structure fixed as suggested.